src/server.ts

```diff
@@ -60,6 +62,7 @@ io.on("connection", (socket) => {
   socket.on("logout", (msg) => logoutHandler.handleMessage(msg));
   socket.on("start", (msg) => startHandler.handleMessage(msg));
   socket.on("disconnect", (msg) => disconnectHandler.handleMessage(msg));
+  socket.on("spell_1_cast", (msg) => spellHandler.handleMessage(msg));
```

Comment:
```suggestion
  socket.on("cast:spell", (msg) => spellHandler.handleMessage(msg));
```

Like that we could also write `cast:trap` for the case traps